tests/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ if(NOT BUILD_DYNAMIC_LINK_PYTHON)
 endif()
 
 
-add_executable(Tests_run)
+add_executable(Tests_run src/Base/Unit.cpp src/Base/Quantity.cpp)
```

Comment:
Why this line? The source files are already added in tests/src/Base/CMakeLists.txt